api/src/main/java/org/openmrs/module/idgen/service/IdentifierSourceService.java

```diff
@@ -265,6 +265,14 @@
      */
     @Transactional(readOnly=true)
     IdentifierSource getIdentifierSourceByUuid(String uuid);
+    
+    /**

```

Comment:
same here

Reply:
Yes this specific file is very well commented.